### PR TITLE
Fix start bug and improve error handling in walkingpad integration

### PR DESCRIPTION
**Pull request by dewgenenny**
Hey, a bug fix for the start command, when the device was turned off. Additionally corrected the walking_time return, as it should have been a timedelta and it was returning an int. 

Also added some better error handling for functions to deal with when the device is turned off.

### CODE REVIEW

**Comment by dewgenenny** [comment]
I am guessing the failed build is pipeline instability again :/

File: miio/walkingpad.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ class WalkingpadStatus(DeviceStatus):
      'sp': 3.0,
      'start_speed': 3.0,
      'step': 180,
-     'time': 121}
+     'walking_time': 121}
```
Comment (rytilahti) [inline miio/walkingpad.py:44]: The code below uses `self.data["time"]` for `walking_time`. Which is the correct input?

**Review by rytilahti — COMMENTED** [review]
The windows builds on azure pipelines have been flaky for no known reason, so I restarted the build manually.

File: miio/walkingpad.py (hunk 1 above)
Comment (dewgenenny) [reply miio/walkingpad.py:44]: good catch, you're right it should stay as time. I have corrected.

**Comment by dewgenenny** [comment]
Should be good to go now

**Comment by rytilahti** [comment]
Great, thanks! :+1:

**Pull request merged**